app/javascript/components/file-uploader.vue

```diff
@@ -15,6 +25,10 @@ export default {
     'dropzone': vue2Dropzone
   },
   props: {
+    infoString: {
+      type: String,
+      default: 'Drag and drop PDFs here to upload'
```

Comment:
suggestion: Since this is a general purpose component, I wonder if it would be better to make `Drag and drop files here to upload` the default and set `Drag and drop PDFs here to upload` for ocr_requests.

Reply:
Done. Take another look when you get a chance?